contrib/fcitx5/template.py

```diff
@@ -0,0 +1,55 @@
+pkgname = "fcitx5"
+pkgver = "5.1.9"
+pkgrel = 0
+build_style = "cmake"
+configure_args = ["-DENABLE_EMOJI=OFF", "-DENABLE_X11=OFF"]
```

Comment:
is there a reason x11 (and emoji) is disabled?

Reply:
None other than me being too lazy to package the required dependencies and the fact that they aren't required for what I wanted fcitx for.

Reply:
i guess you'd need xcb-imdkit, i'll add that

Reply:
added

emoji also seems to build but you'd know more than me

Reply:
@nekopsykose xcb-imdkit is part of my wezterm pr on #1670

Reply:
> emoji also seems to build but you'd know more than me

Indeed, I think I read some outdated information somewhere. It used to required additional dependencies but no longer does it seems: https://github.com/fcitx/fcitx5/commit/fb56879d0fb05984656fe4a103f3a6fabe37c7fb

Thanks for taking care of xcb-imdkit, I've pushed a new version with X11 and emoji support enabled.